### PR TITLE
User-friendlied the layout of parse_file.html. NOTE: "required" input fields only work for "username".

### PR DESCRIPTION
**Pull request by jmdkastro**

### CODE REVIEW

**Comment by keflavich** [comment]
Could you provide a screenshot of this?

For the `required` bug, I think it _does_ work for others, but the `Submit` button will turn green after the username field is filled

**Comment by jmdkastro** [comment]
parse_file

File: templates/parse_file.html [hunk 1]
```diff
@@ -72,17 +72,22 @@ <h1 class="title">CAMELOT Parse file {{filename}}</h1>
...
-                <tr><td colspan=3 class="middle">  <input type="text" id="email" name="Email" value="" required> Your email address <font color="#BDBDBD">(this will be kept private)</font> </td></tr>
-                <tr><td colspan=3 class="middle">  <input type="text" id="adsid" name="adsid" value=""> ADS ID  </td></tr>
-                <tr><td colspan=3 class="middle">  <input type="text" id="doi" name="doi" value="" required> <a href="http://www.doi.org/">DOI: a permanent data link</a> 
-                        or URL <font color="#BDBDBD">(if this is over 64 characters, please <a href="http://tinyurl.com">shorten</a> it
+                <tr><td colspan=3 class="middle"> <input type="text" id="username" name="Username" value="" required> <font color="red">*</font>FirstnameLastname of first author </td></tr>
+                <tr><td colspan=3 class="middle"> <input type="text" id="adsid" name="adsid" value=""> <font color="red">*</font>ADS ID <font color="#BDBDBD">(not the full link)</font> or 
+                                                  <input type="text" id="doi" name="doi" value="" required> <a href="http://www.doi.org/">DOI</a> or URL</td></tr>
```
Comment (keflavich) [inline templates/parse_file.html:85]: Make this "and" since an entry for both is preferred and an entry for DOI/URL is always required.

**Comment by keflavich** [comment]
The "Units" being required is somewhat ambiguous, but I think you should leave it as you've done here.  #91 should include the solution to some units being required.

**Comment by jmdkastro** [comment]
Why would a DOI/URL always be required? An ADS ID should be sufficient, and from that an ADS URL can be generated. Providing an ADS ID and and ADS URL would be redundant I'd think.

**Comment by keflavich** [comment]
It's better to have both if possible, since that provides for a potential validation.

**Comment by jmdkastro** [comment]
As the stupid end-user which I am, I find that highly confusing. In that case, we should simply ask for the ADS URL.

**Pull request merged**